backend/EkoStraz.postman_collection.json

```diff
@@ -0,0 +1,552 @@
+{
+	"info": {
+		"_postman_id": "82e960cd-20c3-42c0-92b7-3161aaf7b317",
+		"name": "EkoStraz",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+	},
+	"item": [
+		{
+			"name": "Interventions",
+			"item": [
+				{
+					"name": "Add Intervention",
+					"request": {
+						"method": "POST",
+						"header": [
+							{
+								"key": "x-functions-key",
+								"value": "abs3adAmZ8qDNgWQqXNAFiPp0cFfXDu6982d9ouyb/TxDiRxWKvWkA==",
```

Comment:
Wydaje mi się, że nie powinniśmy upubliczniać tych kluczy dla 'x-functions-key'.
Napisz Darkowi, żeby sprawdził konto na Azure czy nie ma już długów, bo jak przechwycił to jakiś  bot to mógł narozrabiać chyba? 😎 Te klucze by chyba trzeba teraz unieważnić, i to jak najszybciej, mam rację @dariuszparzygnat czy źle myślę?

Reply:
Z tego co widzę, to 3 różne klucze wyciekły w tym pliku o ile mnie wzrok nie myli :)

Reply:
Poprawione